### PR TITLE
IDE-2902 ignore the sb files

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

**Comment by yanan** [comment]
Test Passed on Jenkins http://ide-build-1:9090/ on this commit 7f59d98ca1c924ba536fd4e436b27630b4c43c21 on build 33

File: tools/plugins/com.liferay.ide.project.ui/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -65,7 +65,8 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.core.jobs,
  org.eclipse.compare,
  org.eclipse.swt,
- org.eclipse.sapphire.platform
+ org.eclipse.sapphire.platform,
+ org.eclipse.wst.jsdt.core
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.ui/META-INF/MANIFEST.MF]: We don't need the jsdt.core dependency.

File: tools/plugins/com.liferay.ide.core/src/com/liferay/ide/core/util/CUCache.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+import java.lang.ref.WeakReference;
+import java.util.Map;
+import java.util.WeakHashMap;
+
+import org.eclipse.jdt.core.JavaCore;
+import org.eclipse.jdt.core.dom.AST;
+import org.eclipse.jdt.core.dom.ASTParser;
+import org.eclipse.jdt.core.dom.CompilationUnit;
+
+public class CUCache
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.core/src/com/liferay/ide/core/util/CUCache.java]: Why do we need this new CUCache?  Why can't we use the previous CUCache?

File: tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrateProjectHandler.java [hunk 3]
```diff
@@ -341,12 +354,66 @@ private boolean shouldAdd( Problem problem )
...
-        if( path.contains( "WEB-INF/classes" ) )
+        if( path.contains( "WEB-INF/classes" ) || path.contains( "WEB-INF/service" ) )
         {
             return false;
         }
 
+        if( path.endsWith( "java" ) )
+        {
+            CompilationUnit ast =
+                CUCache.getCU( problem.getFile(), FileUtil.readContents( problem.getFile() ).toCharArray() );
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrateProjectHandler.java]: Can we use existing CU cache?  Or is that only availble in blade jars?

File: tools/plugins/com.liferay.ide.core/src/com/liferay/ide/core/util/CUCache.java (hunk 2 above)
Comment (gamerson) [inline tools/plugins/com.liferay.ide.core/src/com/liferay/ide/core/util/CUCache.java]: This should be in ide.core.  This should be in the code upgrade tool package.

**Pull request closed without merging**